backend/pom.xml

```diff
@@ -46,6 +46,17 @@
             <version>3.0.3</version>
         </dependency>
 
+        <!-- jdbc for oracle db connection -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
```

Comment:
On startup I get this warning:

```
2022-05-25 12:53:53,884 WARN  [http-nio-9090-exec-9] com.zaxxer.hikari.util.DriverDataSource: Registered driver with driverClassName=oracle.jdbc.driver.OracleDriver was not found, trying direct instantiation.
```

It seems to still work, but the message is concerning. Please test https://stackoverflow.com/a/53430762/201891